src/main/resources/db/migration/V1__imrt_ddl.sql

```diff
@@ -0,0 +1,70 @@
+/* Primary table containing all types of items and the json data from IAT */
+CREATE TABLE item (
+  key             SERIAL PRIMARY KEY, -- Primary key for items, auto-incrementing (change serial to AUTO_INCREMENT for mySQL)
+  id              VARCHAR(9)  NOT NULL, -- Item ID as specified by IAT
```

Comment:
fyi, this isn't specified by IAT.  It is just the item id.  I know this is a nit, but IAT is not the only way that items are created and items still have an item id.

Reply:
And until we get some max values I'd remove all restrictions for now.